internalsdk/ios/ios.go

```diff
@@ -185,11 +185,11 @@ func Client(packetsOut Writer, udpDialer UDPDialer, memChecker MemChecker, confi
 }
 
 func (c *iosClient) start() (ClientWriter, error) {
-	if err := c.loadUserConfig(); err != nil {
-		return nil, log.Errorf("error loading user config: %v", err)
-	}
+	// if err := c.loadUserConfig(); err != nil {
```

Comment:
@atavism So, I did bit of debugging on Exntreiosn why it was not working, It came out that it was throwing an error in the **empty user config**, I guess that was an old mechanism. So I removed now, at the latest we are not getting errors while starting the flashlight.

Reply:
It is very stage without this fix, proxy work you and not for me, Did you test with fresh install or old one?(I guess id its old it should be using old file)

Reply:
Sorry for the delay: I tested with a fresh install